src/api/mod.rs

```diff
@@ -725,18 +762,26 @@ impl<T: Pixel> Context<T> {
     self.inner.send_frame(frame, params)
   }
 
-  /// Retrieve the first-pass data of a two-pass encode for the frame that was
-  /// just encoded. This should be called BEFORE every call to receive_packet()
-  /// (including the very first one), even if no packet was produced by the
-  /// last call to receive_packet, if any (i.e., *EncoderStatus::Encoded* was
-  /// returned). It needs to be called once more after
-  /// *EncoderStatus::LimitReached* is returned, to retrieve the header that
+  /// Returns the first-pass data of a two-pass encode for the frame that was
+  /// just encoded.
+  ///
+  /// This should be called BEFORE every call to [`receive_packet`] (including
+  /// the very first one), even if no packet was produced by the last call to
+  /// [`receive_packet`], if any (i.e., [`EncoderStatus::Encoded`] was
+  /// returned).  It needs to be called once more after
+  /// [`EncoderStatus::LimitReached`] is returned, to retrieve the header that
   /// should be written to the front of the stats file (overwriting the
   /// placeholder header that was emitted at the start of encoding).
   ///
-  /// It is still safe to call this function when receive_packet() returns any
-  /// other error. It will return *None* instead of returning a duplicate copy
-  /// of the previous frame's data.
+  /// It is still safe to call this function when [`receive_packet`] returns
+  /// any other error. It will return `None` instead of returning a duplicate
+  /// copy of the previous frame's data.
+  ///
+  /// [`receive_packet`]: #method.receive_packet
+  /// [`EncoderStatus::Encoded`]: enum.EncoderStatus.html#variant.Encoded
+  /// [`EncoderStatus::LimitReached`]:
```

Comment:
The documentation references these statuses being returned by receive_packet, not this function.